deployment/kube/README.md

```diff
@@ -0,0 +1,22 @@
+# Kubernetes deployments
```

Comment:
please add examples for fetching the ip and address of the server and how to execute orion queries using curl command.